modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -1185,6 +1186,113 @@ definition {
+
+		AppBuilderAdmin.add();
+
+		TableViewBuilder.setName(tableViewName = "My Table View");
+
+		TableViewBuilder.addColumnByDoubleClick(fieldType = "Text");
+
+		TableViewBuilder.addColumnByDoubleClick(fieldType = "Select from List");
+
+		TableViewBuilder.save();
+
+		FormViewBuilder.switchToFormViewTab();
+
+		AppBuilderAdmin.openItem(itemName = "My Form View");
+
+		FormViewBuilder.fieldEllipsisDelete(fieldLabel = "Text");
+
+		FormViewBuilder.validateDeleteUsedObjectMessage();
+
+		FormViewBuilder.validateUsedFormView(formViewName = "My Form View");
+
+		FormViewBuilder.validateUsedTableView(tableViewName = "My Table View");
+
+		FormViewBuilder.deleteButtonInDeleteFromObject();
+
+		FormViewBuilder.validateNoFieldLabel(fieldLabel = "Text");
+		FormViewBuilder.validateNoObjectFieldName(fieldName = "Text");
+
+		FormViewBuilder.deleteFromObjectViaX(fieldName = "Select from List");
+
+		FormViewBuilder.validateDeleteUsedObjectMessage();
+
+		FormViewBuilder.validateUsedFormView(formViewName = "My Form View");
+
+		FormViewBuilder.validateUsedTableView(tableViewName = "My Table View");
+
+		FormViewBuilder.deleteButtonInDeleteFromObject();
+
+		FormViewBuilder.validateNoFieldLabel(fieldLabel = "Select from List");
+		FormViewBuilder.validateNoObjectFieldName(fieldName = "Select from List");
+
+		FormViewBuilder.addFieldByDoubleClick(fieldType = "Numeric");
+
+		FormViewBuilder.fieldEllipsisDelete(fieldLabel = "Numeric");
+
+		FormViewBuilder.validateDeleteObjectMessage();
+
+		FormViewBuilder.deleteButtonInDeleteFromObject();
+
+		FormViewBuilder.validateNoFieldLabel(fieldLabel = "Numeric");
+		FormViewBuilder.validateNoObjectFieldName(fieldName = "Numeric");
+
+		FormViewBuilder.addFieldByDoubleClick(fieldType = "Multiple Selection");
+
+		FormViewBuilder.deleteFromObjectViaX(fieldName = "Multiple Selection");
+
+		FormViewBuilder.validateDeleteObjectMessage();
+
+		FormViewBuilder.cancelButtonInDeleteFromObject();
+
+		FormViewBuilder.validateFieldLabel(fieldLabel = "Multiple Selection");
+		FormViewBuilder.validateObjectFieldName(fieldName = "Multiple Selection");
+
+		FormViewBuilder.deleteFromObjectViaX(fieldName = "Multiple Selection");
+
+		FormViewBuilder.validateDeleteObjectMessage();
+
+		FormViewBuilder.deleteButtonInDeleteFromObject();
+
+		FormViewBuilder.validateNoFieldLabel(fieldLabel = "Multiple Selection");
+		FormViewBuilder.validateNoObjectFieldName(fieldName = "Multiple Selection");
+
+		FormViewBuilder.addFieldByDoubleClick(fieldType = "Single Selection");
+
+		FormViewBuilder.save();
+
+		AppBuilderAdmin.switchTab(tabName = "Table Views");
+
+		AppBuilderAdmin.openItem(itemName = "My Table View");
+
+		TableViewBuilder.validateNoColumnLabel(columnLabel = "Text");
+		TableViewBuilder.validateNoColumnLabelAtSidebar(columnLabel = "Text");
+
+		TableViewBuilder.validateNoColumnLabel(columnLabel = "Select from List");
+		TableViewBuilder.validateNoColumnLabelAtSidebar(columnLabel = "Select from List");
```

Comment:
Hey Tai, I would suggest you to do an additional validation here:

1. You could also validate that the added field "Single Selection" is available in the Table View.